linux/hci/hci.go

```diff
@@ -411,6 +423,24 @@ func (h *HCI) sktLoop() {
 				_ = logger.Error("skt: %v", err)
 			} else {
 				log.Printf("skt: %v", err)
+				// close current hci socket
+				err = h.Close()
```

Comment:
For symmetry, can we use socket.Close() ? Because `h.Close()` is a function for closing HCI, so it may do more than just close the socket.

```suggestion
				err = h.skt.Close()
```